Dockerfile

```diff
@@ -6,18 +6,15 @@ WORKDIR /usr/src/app
 
 COPY package.json yarn.lock ./
 
-RUN yarn install
-
 COPY . .
 
-# Rename Firebase development configs based on OS
-RUN if [ "$(uname)" == "Darwin" ] || [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then \
-      yarn rename-dev-configs; \
-    else \
-      cp /ios/GoogleService-Info.development.plist /ios/GoogleService-Info.plist && \
-      cp /android/app/google-services.development.js /android/app/google-services.js; \
-    fi
+RUN yarn install --verbose
+
+RUN yarn rename-dev-configs
 
 EXPOSE 8081
 
-CMD ["yarn", "start"]
+# To start the app after building it using `docker build -t better-rail-app .` with docker
+# run: `docker run -it -p 8081:8081 better-rail-app /bin/sh` followed by `yarn start`
```

Comment:
It didn't work with /bin/sh on my machine

```suggestion
# run: `docker run -it -p 8081:8081 better-rail-app yarn start`
```

Reply:
Did it work with `yarn start`?
I kept the /bin/sh so you can manually start and stop the yarn because I couldn't find a proper way to enter Android mode/ios mode without having an interactive shell open

Reply:
Yes it did work, for some reason `/bin/sh` didn't start the shell at all

Reply:
hmmm weird, let me check